bin/commands/runs.js

```diff
@@ -56,10 +63,13 @@ module.exports = function run(args) {
 
     // set the no-wrap
     utils.setNoWrap(bsConfig, args);
+    markBlockEnd('setConfig');
 
     // Validate browserstack.json values and parallels specified via arguments
+    markBlockStart('validateConfig');
```

Comment:
Where are we ending it?

Reply:
thanks for pointing out. adding markBlockEnd for this.

Reply:
this is done in latest commit